functions/Remove-DbaDbBackupRestoreHistory.ps1

```diff
@@ -0,0 +1,128 @@
+        The target SQL Server instance or instances.You must have sysadmin access and server version must be SQL Server version 2000 or higher.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Windows and SQL Authentication supported. Accepts credential objects (Get-Credential)
+
+    .PARAMETER KeepDays
+        The number of days of history to keep. Defaults to 30 days.
+
+    .PARAMETER Database
+        The database(s) to process. If unspecified, all databases will be processed.
+
+    .PARAMETER InputObject
+        A collection of databases (such as returned by Get-DbaDatabase).
+
+    .PARAMETER WhatIf
+        Shows what would happen if the command were to run. No actions are actually performed.
+
+    .PARAMETER Confirm
+        Prompts you for confirmation before executing any changing operations within the command.
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: Delete
+        Author: IJeb Reitsma
+
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Remove-DbaDbBackupRestoreHistory
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016
+
+        Remove backup and restore history on SQL Server sql2016 older than 30 days (default period)
+
+        PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -KeepDays 100
+
+        Remove backup and restore history on SQL Server sql2016 older than 100 days
+
+        PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -Database db1
+
+        Remove complete backup and restore history for database db1 on SQL Server sql2016 
+
+        PS C:\> Get-DbaDatabase -SqlInstance sql2016 | Remove-DbaDbBackupRestoreHistory -WhatIf
+
+        Remove complete backup and restore history for all databases on SQL Server sql2016 
+#>
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [parameter(Mandatory, ParameterSetName = "instance")]
+        [parameter(Mandatory, ParameterSetName = "instancedatabase")]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [parameter(Mandatory = $false)]
+        [Alias("Credential")]
+        [PSCredential]
+        $SqlCredential,
+        [parameter(Mandatory = $false, ParameterSetName = "instance")]
+        $KeepDays = 30,
+        [parameter(Mandatory = $false, ParameterSetName = "instancedatabase")]
+        [Alias("Databases")]
+        [object[]]$Database,
```

Comment:
Everything looks great! The only thing I'd change is minor. We used to do `[object[]]$Database,` but have switched to `[string[]]$Database,`